Submitted By:            Wayne Blaszczyk <wblaszcz at bigpond dot net dot au>
Date:                    2013-08.17
Initial Package Version: 4.1.0.4
Upstream Status:         Unknown
Origin:                  Gentoo
Description:             Fixes building when using system Neon-0.30.0

--- a/ucb/source/ucp/webdav-neon/NeonSession.cxx
+++ b/ucb/source/ucp/webdav-neon/NeonSession.cxx
@@ -228,9 +228,6 @@
 }
 
 extern "C" int NeonSession_NeonAuth( void *       inUserData,
-#ifdef NE_FEATURE_SSPI
-                                     const char * inAuthProtocol,
-#endif
                                      const char * inRealm,
                                      int          attempt,
                                      char *       inoutUserName,
@@ -297,14 +294,6 @@
 
     bool bCanUseSystemCreds = false;
 
-#ifdef NE_FEATURE_SSPI
-    bCanUseSystemCreds
-        = (attempt == 0) && // avoid endless loops
-          ne_has_support( NE_FEATURE_SSPI ) && // Windows-only feature.
-          ( ( ne_strcasecmp( inAuthProtocol, "NTLM" ) == 0 ) ||
-            ( ne_strcasecmp( inAuthProtocol, "Negotiate" ) == 0 ) );
-#endif
-
     int theRetVal = pListener->authenticate(
                             OUString::createFromAscii( inRealm ),
                             theSession->getHostName(),
